assets/js/backend.js

```diff
@@ -116,5 +116,24 @@
 
 			e.preventDefault();
 		});
+
+		if (typeof hpPointers !== 'undefined') {
+			var pointers = hpPointers.pointers;
+
+			$.each(pointers, function() {
+				var pointer = this[0],
+					options = $.extend(pointer.options, {
+						content: '<h3>' + pointer.title + '</h3><p>' + pointer.content + '</p>',
```

Comment:
hardcoding HTML in JS is not ok, or there's no other way to render the tooltip? If there are no specific required parameters you can use any other common one like pointer.content or pointer.html instead of 2 (title and content).